keyboards/yetis/keymaps/jogme/keymap.c

```diff
@@ -0,0 +1,240 @@
+                                        _______, _______,   _______, _______),
+    [LAYER_NAV] = LAYOUT(
+                      OS_ALT,  KC_BRID, KC_BRIU, KC_VOLU,   KC_NO,   KC_HOME, KC_PGDN, KC_PGUP,
+    KC_NO,   OS_SHFT, OS_CTRL, KC_TAB,  OS_CMD,  KC_VOLD,   KC_DEL,  KC_LEFT, KC_DOWN, KC_UP,  KC_RGHT, KC_END,
+             KC_MS_L, KC_MS_D, KC_MS_U, KC_MS_R, KC_PSCR,   LA_RGB,  KC_CAPS, KC_BTN1, KC_BTN2, KC_BSPC,
+                                        _______, _______,   _______, _______),
+    [LAYER_NUM] = LAYOUT(
+                      KC_5,    KC_3,    KC_1,    KC_9,      KC_8,    KC_0,    KC_2,    KC_4,
+    KC_7,    OS_SHFT, KC_NO,   KC_NO,   OS_CMD,  KC_F11,    KC_F10,  KC_NO,   KC_NO,   KC_NO,   KC_NO,  KC_6,
+             KC_F7,   KC_F5,   KC_F3,   KC_F1,   KC_F9,     KC_F8,   KC_F12,  KC_F2,   KC_F4,   KC_F6,
+                                        _______, _______,   _______, _______),
+    [LAYER_GAM_LED] = LAYOUT(
+                      KC_1,    KC_2,    KC_3,    KC_4,      _______, RGB_SAD, RGB_SAI, RGB_M_P,
+    KC_0,    KC_P,    KC_A,    KC_W,    KC_D,    KC_R,      RGB_HUD, RGB_VAD, RGB_VAI, RGB_HUI, RGB_M_TW, RGB_M_K,
+             OS_CTRL, KC_Q,    KC_S,    KC_E,    KC_F,      LA_RGB,  RGB_TOG, RGB_M_B, RGB_M_T, RGB_M_R,
+                                        OS_ALT,  KC_B,      LA_COLM, LA_QWER),
+};
+
+bool is_oneshot_cancel_key(uint16_t keycode) {
+    switch (keycode) {
+    case LA_NAV:
+        return true;
+    default:
+        return false;
+    }
+}
+
+bool is_oneshot_ignored_key(uint16_t keycode) {
+    switch (keycode) {
+    case LA_SYM:
+    case LA_NAV:
+    case KC_LSFT:
+    case OS_SHFT:
+    case OS_CTRL:
+    case OS_ALT:
+    case OS_CMD:
+        return true;
+    default:
+        return false;
+    }
+}
+
+bool sw_win_active = false;
+bool sw_lang_active = false;
+
+oneshot_state os_shft_state = os_up_unqueued;
+oneshot_state os_ctrl_state = os_up_unqueued;
+oneshot_state os_alt_state = os_up_unqueued;
+oneshot_state os_cmd_state = os_up_unqueued;
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    update_swapper(
+        &sw_win_active, KC_LGUI, KC_TAB, SW_WIN,
+        keycode, record
+    );
+
+    update_oneshot(
+        &os_shft_state, KC_LSFT, OS_SHFT,
+        keycode, record
+    );
+    update_oneshot(
+        &os_ctrl_state, KC_LCTL, OS_CTRL,
+        keycode, record
+    );
+    update_oneshot(
+        &os_alt_state, KC_LALT, OS_ALT,
+        keycode, record
+    );
+    update_oneshot(
+        &os_cmd_state, KC_LCMD, OS_CMD,
+        keycode, record
+    );
+
+    return true;
+}
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+    return update_tri_layer_state(state, LAYER_SYM, LAYER_NAV, LAYER_NUM);
+}
+
+/* oled */
+#ifdef OLED_ENABLE
+static void render_logo(void) {
+    const char yeti_logo [] = {
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
```

Comment:
This looks to be upside down.  The image should be flipped, and rotation set properly.

Reply:
Fixed - hopefully. At least it is not on my test keyboard